src/OrchardCore.Modules/OrchardCore.Forms/Views/Items/ValidationRulePart.Fields.Edit.cshtml

```diff
@@ -0,0 +1,40 @@
+@using OrchardCore.Forms.ViewModels;
+@model ValidationRulePartEditViewModel
+
+<div class="form-group" >
+    <label asp-for="Type">@T["Validation Type"]</label>
+    <select asp-for="Type" class="form-control content-preview-select">
+        <option value="none">@T["None"]</option>
+        <option value="contains">@T["Contains"]</option>
+        <option value="equals">@T["Equals"]</option>
+        <option value="isAfter">@T["IsAfter"]</option>
+        <option value="isBefore">@T["IsBefore"]</option>
+        <option value="isBoolean">@T["IsBoolean"]</option>
+        <option value="isByteLength">@T["IsByteLength"]</option>
+        <option value="isDate">@T["IsDate"]</option>
+        <option value="isDecimal">@T["IsDecimal"]</option>
+        <option value="isDivisibleBy">@T["IsDivisibleBy"]</option>
+        <option value="isEmpty">@T["IsEmpty"]</option>
+        <option value="isFloat">@T["IsFloat"]</option>
+        <option value="isInt">@T["IsInt"]</option>
+        <option value="isJSON">@T["IsJSON"]</option>
+        <option value="isLength">@T["IsLength"]</option>
+        <option value="isNumeric">@T["IsNumeric"]</option>
+        <option value="matches">@T["Matches"]</option>
+
+    </select>
+    <span class="hint">@T["The Validation type."]</span>
+</div>
+
+<div class="form-group" >
+    <label asp-for="Option">@T["Validation Options"]</label>
```

Comment:
It's not clear from the UI how Validation Options should be used. At least automatically populating it with an example for the selected rule (or adding a placeholder text to the input field for it) would help a lot.